### PR TITLE
Write a tree handle into the summary when the gc blob has not changed

### PR DESCRIPTION
**Pull request by tylerbutler**
Merge #10334 into next branch.

### CODE REVIEW

File: packages/runtime/container-runtime/src/containerRuntime.ts [hunk 1]
```diff
@@ -2206,7 +2217,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             wrapSummaryInChannelsTree(summarizeResult);
             pathPartsForChildren = [channelsTreeName];
         }
-        this.addContainerStateToSummary(summarizeResult);
+        this.addContainerStateToSummary(summarizeResult, fullTree, trackState);
```
Comment (tyler-cai-microsoft) [inline packages/runtime/container-runtime/src/containerRuntime.ts:2220]: We might want to pass in the telemetryContext here

Comment (tyler-cai-microsoft) [reply packages/runtime/container-runtime/src/containerRuntime.ts:2220]: Or is this a purposeful oversight?

File: packages/runtime/container-runtime/src/garbageCollection.ts [hunk 2]
```diff
@@ -711,15 +746,19 @@ export class GarbageCollector implements IGarbageCollector {
 
             return gcStats;
         },
-        { end: true, cancel: "error" });
+            { end: true, cancel: "error" });
```
Comment (tyler-cai-microsoft) [inline packages/runtime/container-runtime/src/garbageCollection.ts:749]: Weird spacing

**Review by tyler-cai-microsoft — APPROVED** [review]

File: packages/runtime/container-runtime/src/containerRuntime.ts (hunk 1 above)
Comment (tylerbutler) [reply packages/runtime/container-runtime/src/containerRuntime.ts]: Good catch.

**Pull request merged**